docs/overlay/features/session-stats.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 2
+---
+
+# Session Stats
+
+The session stats feature can allow you to more accurately determine one's skill, free from the bias of early-on final
+deaths. It allows you to display session stats of any set length up to 100 days back, and can show session fkdr, wlr,
+fk/star, et al.
+
+## Setup
+
+:::info
```

Comment:
Don't use [admonitions](https://docusaurus.io/docs/markdown-features/admonitions) if it's part of the steps a user needs to take.